src/mqt/yaqs/digital/digital_tjm.py

```
@@ -151,20 +150,16 @@ def process_layer(dag: DAGCircuit) -> tuple[list[DAGOpNode], list[DAGOpNode], li
 
         if len(node.qargs) == 1:
             single_qubit_nodes.append(node)
-        elif len(node.qargs) == 2:
-            # Group two-qubit gates by even/odd based on the lower qubit index.
-            q0, q1 = node.qargs[0]._index, node.qargs[1]._index  # ruff:ignore[private-member-access]
-            if min(q0, q1) % 2 == 0:
-                even_nodes.append(node)
-            else:
-                odd_nodes.append(node)
+        # Group gates on two or more qubits by even/odd based on the lowest qubit index.
+        elif min(qubit._index for qubit in node.qargs) % 2 == 0:  # ruff:ignore[private-member-access]
```

Ruff (0.16.1)

[warning] Invalid rule code in suppression: private-member-access

Enable `lint.preview` to use rule names

(RUF102)